### PR TITLE
PORTAL-3977: Update screenshots of the helpdesk access

### PR DESCRIPTION
**Pull request by Nuscly**
There is 3 remaining gif on this page.
localhost_8001_articles_manage-support-access html

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ yarn-install-dev: docker-build ## Install JS dependencies with yarn.lock file an
 
 .PHONY: watch
 watch: yarn-install-dev ## Run on http://localhost:8000 the Helpcenter, with auto refresh on file changes
-	$(DOCKER_RUN) --expose=8000 -p=8000:8000 -p=35729:35729 ${DOCKER_IMAGE_TAG} yarn gulp serve
+	$(DOCKER_RUN) --expose=8001 -p=8001:8000 -p=35729:35729 ${DOCKER_IMAGE_TAG} yarn gulp serve
```
Comment (manolo317) [inline Makefile:42]: Do you want to commit it?

**Review by manolo317 — APPROVED** [review]
LGTM with question

**Review by vilain-corentin — APPROVED** [review]
It's ok to use an Akeneo internal profile ?
since "Akeneo Channel Manager" isn't expose ?

**Comment by Nuscly** [comment]
> It's ok to use an Akeneo internal profile ? since "Akeneo Channel Manager" isn't expose ?

Your are right, I will redo the screenshot with the right user.

File: Makefile (hunk 1 above)
Comment (Nuscly) [reply Makefile:42]: Nope, oups 👍🏻

**Pull request merged**